README.md

```diff
@@ -1,12 +1,14 @@
-# Shopsys Project-base
+# Shopsys Framework Demoshop
 
-This repository is part of Shopsys Framework - a **scalable ecommerce framework** for fast-growing ecommerce sites created and maintained by in-house developers or outsourcing agencies.
+This repository is Demoshop project of Shopsys Framework - a **scalable ecommerce framework** for fast-growing ecommerce sites created and maintained by in-house developers or outsourcing agencies.
 
-Shopsys/project-base repository contains the fully ready base for building your Shopsys Framework projects. This repository is READ-ONLY and it is maintained by monorepo [shopsys/shopsys](https://github.com/shopsys/shopsys).
+Shopsys/demoshop repository contains demostration project to show how you can build your own project on top of Shopsys Framework. 
+It started in July of 2018 based on [shopsys/project-base](https://github.com/shopsys/project-base) and since then we added several new features that are not included in Shopsys Framework. 
```

Comment:
I wouldnt even add this sentence. I think that this will only confuse people. I think that sentence after that summerize it all well.